bin/buphalo

```diff
@@ -3,10 +3,26 @@
 
 cd "$(dirname "$0")" || exit
 cd ..
+
+export Neighborhoods_Buphalo_V1_TemplateTree_Map_Builder_FactoryInterface__TemplateTreeDirectoryPaths=default:$PWD/vendor/neighborhoods/buphalo/template-tree/V1,diagnostic:$PWD/vendor/neighborhoods/throwable-diagnostic-component/template-tree/V1,worker:$PWD/template-tree/V1
+export Neighborhoods_Buphalo_V1_FabricationFile_Map_BuilderInterface__FinderFileNames="*.buphalo.v1.fabrication.yml"
+
 # shellcheck disable=SC2034
 Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__SourceDirectoryPath=$PWD/src \
 Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__FabricationDirectoryPath=$PWD/fab \
 Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__NamespacePrefix=Neighborhoods\\KojoWorkerDecoratorComponent\\ \
-Neighborhoods_Buphalo_V1_TemplateTree_Map_Builder_FactoryInterface__TemplateTreeDirectoryPaths=default:$PWD/vendor/neighborhoods/buphalo/template-tree/V1,diagnostic:$PWD/vendor/neighborhoods/throwable-diagnostic-component/template-tree/V1,worker:$PWD/template-tree/V1 \
-Neighborhoods_Buphalo_V1_FabricationFile_Map_BuilderInterface__FinderFileNames="*.buphalo.v1.fabrication.yml" \
-vendor/bin/buphalo
\ No newline at end of file
+vendor/bin/buphalo
+
+# shellcheck disable=SC2034
+Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__SourceDirectoryPath=$PWD/WorkerV1/src \
+Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__FabricationDirectoryPath=$PWD/WorkerV1/fab \
+Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__NamespacePrefix=Neighborhoods\\KojoWorkerDecoratorComponent\\WorkerV1\\ \
+vendor/bin/buphalo
+
+# shellcheck disable=SC2034
+Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__SourceDirectoryPath=$PWD/WorkerDecorators/CrashedThresholdV1/src \
+Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__FabricationDirectoryPath=$PWD/WorkerDecorators/CrashedThresholdV1/fab \
+Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__NamespacePrefix=Neighborhoods\\KojoWorkerDecoratorComponent\\WorkerDecorators\\CrashedThresholdV1\\ \
```

Comment:
Related to the other comment. Would also require changing the file structure a bit.
```suggestion
Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__SourceDirectoryPath=$PWD/WorkerDecorators/src \
Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__FabricationDirectoryPath=$PWD/WorkerDecorators/fab \
Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__NamespacePrefix=Neighborhoods\\KojoWorkerDecoratorComponent\\WorkerDecorators\\ \
```

Reply:
How about I change the script to loop over all subfolders inside WorkerDecorators?

Reply:
I've modified it to loop over all subfolders in WorkerDecorators